README.md

```diff
@@ -23,7 +23,17 @@ The bq_cred.json is service account for Big Query, it needs to be supplied and p
 	student-dashboard-saml.key
 	student-dashboard-saml.pem
 
-## Docker commands for deploying the app
+# Control course view options
+
+View options can be controlled at the global and course level.
```

Comment:
I had a little bit of confusion around these settings, these actually enable/disable the ability to turn on the view at all. You still have to go in and enable/disable the view in the database. 

Maybe just a comment to this point could be added. Like

For each of these view settings set to True you need to configure at the course level which views to enable (by default all views are off)
For each of these view settings set to False that view will be disabled for that view even if enabled via the UI at the course level.